README.md

```diff
@@ -159,6 +159,9 @@ The following resources will be created:
 | work_disk_volume_size | Size of the external EBS volume | string | `100` | no |
 | work_disk_volume_type | Volume type of the external EBS volume | string | `standard` | no |
 | worker_tsa_port | tsa port that the worker can use to connect to the web | string | `2222` | no |
+| teleport_auth_token_encrypted | Teleport server node token encrypted with context teleport=tokens | string | `` | no |
```

Comment:
in the rest of the files its called `teleport_auth_token`

Reply:
yes because that's not encrypted. this one is

Reply:
wrong copy/paste fixing